docs/en/plugin_common_options.md

```diff
@@ -0,0 +1,15 @@
+General Plugin Options
+=================
+
+The following options are available for all plugins. 
+
+* **allow_failures** [bool, optional] - If true, allow the build to succeed even if this plugin fails.
+* **directory** [string, optional] - This option lets you specify the tests directory to run.
```

Comment:
"This option lets you specify the directory this plugin to run on."

Plugin function is not always to test something.